utils/node_local_test.py

```diff
@@ -1136,6 +1140,7 @@ def needs_dfuse(method):
     def _helper(self):
         self.dfuse = DFuse(self.server,
                            self.conf,
+                           caching=False,
```

Comment:
i would prefer to have all tests with caching enabled, and:
needs_dfuse_with_cache
be
needs_dfuse_without_cache ?

Reply:
It's really testing two different things, if we enabled caching then a lot of the tests wouldn't be covering what we'd intended them to.

Additionally, there are failures if you just turn on caching, so we probably want some code to run all tests twice, once with and once without, but skip running the once that require caching without caching.

Part of the issue is rename, with caching a simple test ike the following fails:

touch a
mv a b
stat b

What happens is the mv checks to see if b exists for some reason, puts a negative dentry in the cache then the mv doesn't automatically remove the cache entry so the stat fails with "no such file or directory".  I'm still investigating this further but it seems like it should be a kernel bug.

I'll run some more tests on this but I don't want to loose the coverage of running without caching.